session/session_test.go

```diff
@@ -1996,7 +1996,7 @@ func (s *testSchemaSerialSuite) TestSchemaCheckerSQL(c *C) {
 	tk1.MustExec(`alter table t add index idx3(c);`)
 	tk.MustQuery(`select * from t for update`)
 	_, err = tk.Exec(`commit;`)
-	c.Assert(terror.ErrorEqual(err, domain.ErrInfoSchemaChanged), IsTrue, Commentf("err %v", err))
+	c.Assert(err, NotNil)
```

Comment:
The error is write conflict now?

Reply:
Yes

Reply:
`select * from t where id = 666 for update`
Could this one avoid the conflict error?

Reply:
Yes, using this will get `schema has changed` error